src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -5313,8 +5376,9 @@ protected internal override void Validate()
         public override StringBuilder ToXml(StringBuilder sb)
         {
             Prefix(sb);
-            XmlAttribHex(sb, "TypeID", TypeID);
+            XmlAttrib(sb, "TypeName", TypeName);
```

Comment:
You want the most important/useful fields first (so the default list shows things in the best order).    It is a minor point